azure-pipelines/pull-request-validation/build-consumers.yml

```diff
@@ -227,7 +239,7 @@ stages:
             eval $(dbus-launch --sh-syntax)
             sudo apt install gnome-keyring
             /usr/bin/gnome-keyring-daemon --start --components=secrets
-            ./gradlew LinuxBroker:linuxBrokerUnitTestCoverageReport -Plabtest -PlabSecret=$(LabVaultAppCert) -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}} -PcodeCoverageEnabled=true -Psystemd_mode_enabled=false
+            ./gradlew LinuxBroker:linuxBrokerUnitTestCoverageReport -PcodeCoverageEnabled=true -PlabSecret=$(LabAuth) -Psystemd_mode_enabled=false --build-cache --info
```

Comment:
is -ProbolectricSdkVersion also not needed?

Reply:
No it's not being used in standalone linux pipeline